go.mod

```diff
@@ -3,9 +3,9 @@ module mostwanted.io/galera-health
 go 1.18
```

Comment:
Currently golang is on 1.20.

1.18 is no longer supported, https://endoflife.date/go

This can be changed manually in this file, or use `go mod edit -go=x.xx` to do this automated.